Unique Paths

```diff
@@ -0,0 +1,32 @@
+// Author: Emre Çelik
+// Question Link: https://leetcode.com/problems/unique-paths/
+
+
+/*
+      3*7
+ _ _ _ _ _ _ _
+|_|_|_|_|_|_|_|
+|_|_|_|_|_|_|_|    These matrixes will be filled with number of unique paths.
+|_|_|_|_|_|_|_|
+
+*/
+
+class Solution {
+public:
+    int uniquePaths(int m, int n) {
+        
+        //Initialize ans matrix as a answer matrix.
+        vector<vector<int>> ans(m, vector<int>(n, 1));
+        
+        //We are starting with i = 1 and j = 1 because there are only 1 way to reach borders of the way.
```

Comment:
grammar & sentence refactorization: We start with i = 1 and j = 1 because there is only one way to reach borders of the matrix.